arex-storage-web-api/src/main/java/com/arextest/storage/service/AgentWorkingService.java

```diff
@@ -153,6 +158,30 @@ public <T extends Mocker> byte[] queryMockResult(@NotNull T recordItem,
     return result;
   }
 
+  public byte[] queryAllMocker(String recordId, String[] fieldNames, String[] categoryList) {
+    List<RepositoryProvider<? extends Mocker>> repositoryProviderList = repositoryProviderFactory.getRepositoryProviderList();
+
+    Set<MockCategoryType> categoryTypes = Sets.newHashSetWithExpectedSize(categoryList.length);
+    for (String categoryName : categoryList) {
+      MockCategoryType category = repositoryProviderFactory.findCategory(categoryName);
+      if (category == null) {
+        continue;
+      }
+      categoryTypes.add(category);
+    }
+
+    // find data in the order of rolling -> pinned -> auto pinned
+    for (RepositoryProvider<? extends Mocker> repositoryReader : repositoryProviderList) {
```

Comment:
rolling and pinned recordId generation rules are different,  
can we identify which type based on the recordId? (reduce loop times)

Reply:
It has been discussed privately. It can be optimized later to distinguish the data source through the transparent transmission parameters of the schedule.